app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+class IncorrectCommand(Exception):
+    """Error if command is not move"""
+
+
+def move_file(command: str) -> None:
+    command, copied_file, new_file = command.split()
+    if command != "mv":
+        raise IncorrectCommand("Function supports only mv command")
+
+    if "/" in new_file:
+        folders = new_file.split("/")[:-1]
```

Comment:
Is your code cross-platform ?) For example, will it works on windows with `\` ?)

HINT: use os.path.split()